docs/platforms/flutter/configuration/options.mdx

```diff
@@ -25,7 +25,7 @@ Learn more about [DSN utilization](/product/sentry-basics/dsn-explainer/#dsn-uti
 
 <ConfigKey name="debug">
 
-Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.
+Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is `true` in debug mode, otherwise it is `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.
```

Comment:
```suggestion
Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is `true` if your app runs in [debug mode](https://docs.flutter.dev/testing/build-modes#debug), otherwise it is `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.
```